python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_defs.py

```diff
@@ -559,6 +596,7 @@ def load_assets_from_fivetran_instance(
     poll_interval: float = DEFAULT_POLL_INTERVAL,
     poll_timeout: Optional[float] = None,
     fetch_column_metadata: bool = True,
+    translator: Optional[DagsterFivetranTranslator] = None,
```

Comment:
With the translator pattern on newer integrations, we're formatting this as a `Type[DagsterFivetranTranslator]`, so a user just passes the subclass they've defined rather than an instance.